vertica_backup/backup.py

```diff
@@ -154,10 +158,11 @@ def main(argv=None):
         delete_pickles(fs_store)
         delete_pickles(swift_store, config['retain'])
 
-    except:
+    except Exception as e:
```

Comment:
Minor but e is not used, it could just be "except Exception" You are right it should explicitly have Exception set.